keypair/keypair_test.go

```diff
@@ -0,0 +1,63 @@
+package keypair
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestParsePrivateKey(t *testing.T) {
+	r := require.New(t)
+
+	defer os.Remove("private.pem")
```

Comment:
the problem with this test is that it "leaks" state and might collide with other test that, for example, might also depend on this `private.pem` filepath. 

One way to work around this is to create randomly named temporary files -- https://golang.org/pkg/io/ioutil/#TempFile

Reply:
Ah, forgot about this. Thank you!